src/ctim/schemas/common.cljc

```diff
@@ -20,7 +20,7 @@
             [flanders.predicates :as fp]
             [clojure.string :as str]))
 
-(def ctim-schema-version "1.0.24")
+(def ctim-schema-version "1.1.0")
```

Comment:
these changes introduce incompatibilities without being significant breaking changes.

Reply:
Maybe at some point we could look into a strict application of semver. In the end there are infinite numbers so we won't run out of them :) i.e. it's simpler to be strict/pessimistic about what constitutes a breaking change

For example, https://github.com/threatgrid/ctia/pull/1065 requires a somewhat delicate migration plan. That would be even clearer if its ctim change moved from 1.* to 2.*

(in that specific PR it doesn't matter a lot since it already says `do not merge`, but maybe in a future, newer PRs could make a mistake)

Reply:
@gbuisson wdyt?

Reply:
for 2.0.0 I would expect major changes, including for instance support for Stix 3.0, a mali output or something like that, a 1.5.0 sounds more reasonable... but I agree it's completely psychological.

Reply:
In the CTIM library, the `ctim-schema-version` should only change for schema changes, it means that even an increment in the patch version means that a breaking change like a new optional field has been added. We were used to increment only the patch or the minor version depending on the number of changes, if new entities were added. `1.1.0` looks good regarding this change. I'm open to follow another rules for the schema versioning of CTIM